plugin.video.mms/resources/lib/fsutils.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015 Thomas Amland
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+from collections import namedtuple
+
+try:
```

Comment:
I don't really understand this try-except: 

1. This won't except, as `xbmc` and `xbmcvfs` are always there on Kodi Matrix
1. If somehow it does fail, `xbmc=None` and line 31 will cause an exception. 

So this makes little sense.

Reply:
Fair point. I tried to keep the original version, with a thought that's it's used somehow outside of kodi :) Fixing.